Installation problem on R 4.0.2

I'm having an issue installing on R 4.0.2. Was wondering if this is an issue with the new rtools.

I have rtools and the necessary C++. I'm able to use rstan for instance which requires the setup. 

<details>
  <summary>Long installation message with error</summary>

```r
> devtools::install_github("fnoorian/quadprogpp")
Downloading GitHub repo fnoorian/quadprogpp@HEAD
√  checking for file 'C:\Users\spinkney\AppData\Local\Temp\RtmpCKEsi0\remotes69bc32156e12\fnoorian-quadprogpp-797627b/DESCRIPTION' (441ms)
-  preparing 'quadprogpp':
√  checking DESCRIPTION meta-information ... 
-  cleaning src
-  checking for LF line-endings in source and make files and shell scripts (1.4s)
-  checking for empty or unneeded directories
-  building 'quadprogpp_1.1-0.tar.gz'
   
'\\csiadsrm01\Public\FTP Survey 2007\Smart Control\spinkney\Overlapping Samples\bayesian overlap'
CMD.EXE was started with the above path as the current directory.
UNC paths are not supported.  Defaulting to Windows directory.
* installing *source* package 'quadprogpp' ...
** using staged installation
** libs

*** arch - i386
"C:/rtools40/mingw32/bin/"g++ -std=gnu++11  -I"C:/R/R-40~1.2/include" -DNDEBUG  -I'C:/R/R-4.0.2/library/Rcpp/include' -I'C:/R/R-4.0.2/library/RcppEigen/include'      -include C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp -I "C:/R/R-4.0.2/library/StanHeaders/include" -I "C:/R/R-4.0.2/library/RcppEigen/include"   -O2 -Wall  -mfpmath=sse -msse2 -mstackrealign -c RcppExports.cpp -o RcppExports.o
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:392,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:60:39: warning: ignoring attributes on template argument '__m128' {aka '__vector(4) float'} [-Wignored-attributes]
 template<> struct is_arithmetic<__m128>  { enum { value = true }; };
                                       ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:61:40: warning: ignoring attributes on template argument '__m128i' {aka '__vector(2) long long int'} [-Wignored-attributes]
 template<> struct is_arithmetic<__m128i> { enum { value = true }; };
                                        ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:62:40: warning: ignoring attributes on template argument '__m128d' {aka '__vector(2) double'} [-Wignored-attributes]
 template<> struct is_arithmetic<__m128d> { enum { value = true }; };
                                        ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:161:43: warning: ignoring attributes on template argument 'Eigen::internal::Packet4f' {aka '__vector(4) float'} [-Wignored-attributes]
 template<> struct unpacket_traits<Packet4f> { typedef float  type; enum {size=4, alignment=Aligned16}; typedef Packet4f half; };
                                           ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:162:43: warning: ignoring attributes on template argument 'Eigen::internal::Packet2d' {aka '__vector(2) double'} [-Wignored-attributes]
 template<> struct unpacket_traits<Packet2d> { typedef double type; enum {size=2, alignment=Aligned16}; typedef Packet2d half; };
                                           ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:163:43: warning: ignoring attributes on template argument 'Eigen::internal::Packet4i' {aka '__vector(2) long long int'} [-Wignored-attributes]
 template<> struct unpacket_traits<Packet4i> { typedef int    type; enum {size=4, alignment=Aligned16}; typedef Packet4i half; };
                                           ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:718:35: warning: ignoring attributes on template argument 'Eigen::internal::Packet4f' {aka '__vector(4) float'} [-Wignored-attributes]
 struct palign_impl<Offset,Packet4f>
                                   ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:741:35: warning: ignoring attributes on template argument 'Eigen::internal::Packet4i' {aka '__vector(2) long long int'} [-Wignored-attributes]
 struct palign_impl<Offset,Packet4i>
                                   ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:764:35: warning: ignoring attributes on template argument 'Eigen::internal::Packet2d' {aka '__vector(2) double'} [-Wignored-attributes]
 struct palign_impl<Offset,Packet2d>
                                   ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:778:34: warning: ignoring attributes on template argument 'Eigen::internal::Packet4f' {aka '__vector(4) float'} [-Wignored-attributes]
 ptranspose(PacketBlock<Packet4f,4>& kernel) {
                                  ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:783:34: warning: ignoring attributes on template argument 'Eigen::internal::Packet2d' {aka '__vector(2) double'} [-Wignored-attributes]
 ptranspose(PacketBlock<Packet2d,2>& kernel) {
                                  ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/PacketMath.h:790:34: warning: ignoring attributes on template argument 'Eigen::internal::Packet4i' {aka '__vector(2) long long int'} [-Wignored-attributes]
 ptranspose(PacketBlock<Packet4i,4>& kernel) {
                                  ^
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:374,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/Default/ConjHelper.h:15:70: warning: ignoring attributes on template argument 'Eigen::internal::Packet4f' {aka '__vector(4) float'} [-Wignored-attributes]
   template<> struct conj_helper<PACKET_REAL, PACKET_CPLX, false,false> {                                          \
                                                                      ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/Complex.h:232:1: note: in expansion of macro 'EIGEN_MAKE_CONJ_HELPER_CPLX_REAL'
 EIGEN_MAKE_CONJ_HELPER_CPLX_REAL(Packet2cf,Packet4f)
 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/Default/ConjHelper.h:22:70: warning: ignoring attributes on template argument 'Eigen::internal::Packet4f' {aka '__vector(4) float'} [-Wignored-attributes]
   template<> struct conj_helper<PACKET_CPLX, PACKET_REAL, false,false> {                                          \
                                                                      ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/Complex.h:232:1: note: in expansion of macro 'EIGEN_MAKE_CONJ_HELPER_CPLX_REAL'
 EIGEN_MAKE_CONJ_HELPER_CPLX_REAL(Packet2cf,Packet4f)
 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/Default/ConjHelper.h:15:70: warning: ignoring attributes on template argument 'Eigen::internal::Packet2d' {aka '__vector(2) double'} [-Wignored-attributes]
   template<> struct conj_helper<PACKET_REAL, PACKET_CPLX, false,false> {                                          \
                                                                      ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/Complex.h:417:1: note: in expansion of macro 'EIGEN_MAKE_CONJ_HELPER_CPLX_REAL'
 EIGEN_MAKE_CONJ_HELPER_CPLX_REAL(Packet1cd,Packet2d)
 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/Default/ConjHelper.h:22:70: warning: ignoring attributes on template argument 'Eigen::internal::Packet2d' {aka '__vector(2) double'} [-Wignored-attributes]
   template<> struct conj_helper<PACKET_CPLX, PACKET_REAL, false,false> {                                          \
                                                                      ^
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/arch/SSE/Complex.h:417:1: note: in expansion of macro 'EIGEN_MAKE_CONJ_HELPER_CPLX_REAL'
 EIGEN_MAKE_CONJ_HELPER_CPLX_REAL(Packet1cd,Packet2d)
 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/MatrixBase.h:130,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:436,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:22:30: error: 'conditional_t' in namespace 'std' does not name a template type
 using double_return_t = std::conditional_t<std::is_const<std::remove_reference_t<T>>::value,
                              ^~~~~~~~~~~~~
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:22:25: note: suggested alternative: 'conditional'
 using double_return_t = std::conditional_t<std::is_const<std::remove_reference_t<T>>::value,
                         ^~~
                         conditional
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:26:31: error: 'conditional_t' in namespace 'std' does not name a template type
 using reverse_return_t = std::conditional_t<std::is_const<std::remove_reference_t<T>>::value,
                               ^~~~~~~~~~~~~
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:26:26: note: suggested alternative: 'conditional'
 using reverse_return_t = std::conditional_t<std::is_const<std::remove_reference_t<T>>::value,
                          ^~~
                          conditional
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:31:28: error: 'conditional_t' in namespace 'std' does not name a template type
 using vari_return_t = std::conditional_t<std::is_const<std::remove_reference_t<T>>::value,
                            ^~~~~~~~~~~~~
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:31:23: note: suggested alternative: 'conditional'
 using vari_return_t = std::conditional_t<std::is_const<std::remove_reference_t<T>>::value,
                       ^~~
                       conditional
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:36:31: error: 'conditional_t' in namespace 'std' does not name a template type
 using forward_return_t = std::conditional_t<std::is_const<std::remove_reference_t<T>>::value,
                               ^~~~~~~~~~~~~
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:36:26: note: suggested alternative: 'conditional'
 using forward_return_t = std::conditional_t<std::is_const<std::remove_reference_t<T>>::value,
                          ^~~
                          conditional
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:60:10: error: 'enable_if_t' in namespace 'std' does not name a template type
     std::enable_if_t<std::is_pointer<T>::value, reverse_return_t<T>>
          ^~~~~~~~~~~
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:60:5: note: suggested alternative: 'enable_if'
     std::enable_if_t<std::is_pointer<T>::value, reverse_return_t<T>>
     ^~~
     enable_if
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:66:10: error: 'enable_if_t' in namespace 'std' does not name a template type
     std::enable_if_t<(!std::is_pointer<T>::value && !is_fvar<T>::value
          ^~~~~~~~~~~
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:66:5: note: suggested alternative: 'enable_if'
     std::enable_if_t<(!std::is_pointer<T>::value && !is_fvar<T>::value
     ^~~
     enable_if
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:73:10: error: 'enable_if_t' in namespace 'std' does not name a template type
     std::enable_if_t<is_fvar<T>::value, forward_return_t<T>>
          ^~~~~~~~~~~
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:73:5: note: suggested alternative: 'enable_if'
     std::enable_if_t<is_fvar<T>::value, forward_return_t<T>>
     ^~~
     enable_if
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:79:10: error: 'enable_if_t' in namespace 'std' does not name a template type
     std::enable_if_t<std::is_arithmetic<T>::value, double_return_t<T>>
          ^~~~~~~~~~~
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:79:5: note: suggested alternative: 'enable_if'
     std::enable_if_t<std::is_arithmetic<T>::value, double_return_t<T>>
     ^~~
     enable_if
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:117:5: error: 'forward_return_t' does not name a type; did you mean 'error_return'?
     forward_return_t<T> operator()(T &v) const { return v.d_; }
     ^~~~~~~~~~~~~~~~
     error_return
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:148:10: error: 'enable_if_t' in namespace 'std' does not name a template type
     std::enable_if_t<std::is_pointer<T>::value, reverse_return_t<T>>
          ^~~~~~~~~~~
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:148:5: note: suggested alternative: 'enable_if'
     std::enable_if_t<std::is_pointer<T>::value, reverse_return_t<T>>
     ^~~
     enable_if
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:154:10: error: 'enable_if_t' in namespace 'std' does not name a template type
     std::enable_if_t<!std::is_pointer<T>::value, reverse_return_t<T>>
          ^~~~~~~~~~~
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:154:5: note: suggested alternative: 'enable_if'
     std::enable_if_t<!std::is_pointer<T>::value, reverse_return_t<T>>
     ^~~
     enable_if
C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/eigen_plugins.h:183:5: error: 'vari_return_t' does not name a type; did you mean 'error_return'?
     vari_return_t<T> operator()(T &v) const { return v.vi_; }
     ^~~~~~~~~~~~~
     error_return
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:367,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h: In instantiation of 'struct Eigen::internal::find_best_packet<int, -1>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Matrix.h:22:57:   required from 'struct Eigen::internal::traits<Eigen::Matrix<int, 1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Array.h:17:8:   required from 'struct Eigen::internal::traits<Eigen::Array<int, 1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/PlainObjectBase.h:98:7:   required from 'class Eigen::PlainObjectBase<Eigen::Array<int, 1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Array.h:45:7:   required from 'class Eigen::Array<int, 1, -1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/SVD/BDCSVD.h:101:22:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:44: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<int>::type' {aka '__vector(2) long long int'} [-Wignored-attributes]
          bool Stop = Size==Dynamic || (Size%unpacket_traits<PacketType>::size)==0 || is_same<PacketType,typename unpacket_traits<PacketType>::half>::value>
                                       ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<int>::type' {aka '__vector(2) long long int'} [-Wignored-attributes]
          bool Stop = Size==Dynamic || (Size%unpacket_traits<PacketType>::size)==0 || is_same<PacketType,typename unpacket_traits<PacketType>::half>::value>
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<int>::type' {aka '__vector(2) long long int'} [-Wignored-attributes]
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::unpacket_traits<__vector(2) long long int>::half' {aka '__vector(2) long long int'} [-Wignored-attributes]
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:188:88: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<int>::type' {aka '__vector(2) long long int'} [-Wignored-attributes]
   typedef typename find_best_packet_helper<Size,typename packet_traits<T>::type>::type type;
                                                                                        ^~~~
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:434,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h: In instantiation of 'class Eigen::DenseCoeffsBase<Eigen::Array<int, 1, -1>, 0>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:300:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Array<int, 1, -1>, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:551:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Array<int, 1, -1>, 3>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseBase.h:41:34:   required from 'class Eigen::DenseBase<Eigen::Array<int, 1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/ArrayBase.h:39:34:   required from 'class Eigen::ArrayBase<Eigen::Array<int, 1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/PlainObjectBase.h:98:7:   required from 'class Eigen::PlainObjectBase<Eigen::Array<int, 1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Array.h:45:7:   required from 'class Eigen::Array<int, 1, -1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/SVD/BDCSVD.h:101:22:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:55:30: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<int>::type' {aka '__vector(2) long long int'} [-Wignored-attributes]
                      >::type PacketReturnType;
                              ^~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h: In instantiation of 'class Eigen::DenseCoeffsBase<Eigen::Ref<Eigen::Array<int, 1, -1> >, 0>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:300:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Ref<Eigen::Array<int, 1, -1> >, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:551:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Ref<Eigen::Array<int, 1, -1> >, 3>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseBase.h:41:34:   required from 'class Eigen::DenseBase<Eigen::Ref<Eigen::Array<int, 1, -1> > >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/ArrayBase.h:39:34:   required from 'class Eigen::ArrayBase<Eigen::Ref<Eigen::Array<int, 1, -1> > >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/MapBase.h:37:34:   required from 'class Eigen::MapBase<Eigen::Ref<Eigen::Array<int, 1, -1> >, 0>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/MapBase.h:219:34:   required from 'class Eigen::MapBase<Eigen::Ref<Eigen::Array<int, 1, -1> >, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Ref.h:58:34:   required from 'class Eigen::RefBase<Eigen::Ref<Eigen::Array<int, 1, -1> > >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Ref.h:192:76:   required from 'class Eigen::Ref<Eigen::Array<int, 1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/SVD/BDCSVD.h:694:17:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:55:30: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<int>::type' {aka '__vector(2) long long int'} [-Wignored-attributes]
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:367,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h: In instantiation of 'struct Eigen::internal::find_best_packet<float, 4>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Matrix.h:22:57:   required from 'struct Eigen::internal::traits<Eigen::Matrix<float, 4, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Geometry/Quaternion.h:224:7:   required from 'struct Eigen::internal::traits<Eigen::Quaternion<float> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Geometry/arch/Geometry_SSE.h:24:46:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:44: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<float>::type' {aka '__vector(4) float'} [-Wignored-attributes]
          bool Stop = Size==Dynamic || (Size%unpacket_traits<PacketType>::size)==0 || is_same<PacketType,typename unpacket_traits<PacketType>::half>::value>
                                       ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<float>::type' {aka '__vector(4) float'} [-Wignored-attributes]
          bool Stop = Size==Dynamic || (Size%unpacket_traits<PacketType>::size)==0 || is_same<PacketType,typename unpacket_traits<PacketType>::half>::value>
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<float>::type' {aka '__vector(4) float'} [-Wignored-attributes]
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::unpacket_traits<__vector(4) float>::half' {aka '__vector(4) float'} [-Wignored-attributes]
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:188:88: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<float>::type' {aka '__vector(4) float'} [-Wignored-attributes]
   typedef typename find_best_packet_helper<Size,typename packet_traits<T>::type>::type type;
                                                                                        ^~~~
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:434,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h: In instantiation of 'class Eigen::DenseCoeffsBase<Eigen::Matrix<float, 4, 1>, 0>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:300:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<float, 4, 1>, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:551:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<float, 4, 1>, 3>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseBase.h:41:34:   required from 'class Eigen::DenseBase<Eigen::Matrix<float, 4, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/MatrixBase.h:48:34:   required from 'class Eigen::MatrixBase<Eigen::Matrix<float, 4, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/PlainObjectBase.h:98:7:   required from 'class Eigen::PlainObjectBase<Eigen::Matrix<float, 4, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Matrix.h:178:7:   required from 'class Eigen::Matrix<float, 4, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Geometry/Quaternion.h:46:50:   required from 'class Eigen::QuaternionBase<Eigen::Quaternion<float> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Geometry/Quaternion.h:232:7:   required from 'class Eigen::Quaternion<float>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Geometry/arch/Geometry_SSE.h:27:3:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:55:30: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<float>::type' {aka '__vector(4) float'} [-Wignored-attributes]
                      >::type PacketReturnType;
                              ^~~~~~~~~~~~~~~~
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:367,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h: In instantiation of 'struct Eigen::internal::find_best_packet<double, 4>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Matrix.h:22:57:   required from 'struct Eigen::internal::traits<Eigen::Matrix<double, 4, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Geometry/Quaternion.h:224:7:   required from 'struct Eigen::internal::traits<Eigen::Quaternion<double> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Geometry/arch/Geometry_SSE.h:88:47:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:44: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
          bool Stop = Size==Dynamic || (Size%unpacket_traits<PacketType>::size)==0 || is_same<PacketType,typename unpacket_traits<PacketType>::half>::value>
                                       ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
          bool Stop = Size==Dynamic || (Size%unpacket_traits<PacketType>::size)==0 || is_same<PacketType,typename unpacket_traits<PacketType>::half>::value>
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::unpacket_traits<__vector(2) double>::half' {aka '__vector(2) double'} [-Wignored-attributes]
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:188:88: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
   typedef typename find_best_packet_helper<Size,typename packet_traits<T>::type>::type type;
                                                                                        ^~~~
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:434,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h: In instantiation of 'class Eigen::DenseCoeffsBase<Eigen::Matrix<double, 4, 1>, 0>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:300:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<double, 4, 1>, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:551:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<double, 4, 1>, 3>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseBase.h:41:34:   required from 'class Eigen::DenseBase<Eigen::Matrix<double, 4, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/MatrixBase.h:48:34:   required from 'class Eigen::MatrixBase<Eigen::Matrix<double, 4, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/PlainObjectBase.h:98:7:   required from 'class Eigen::PlainObjectBase<Eigen::Matrix<double, 4, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Matrix.h:178:7:   required from 'class Eigen::Matrix<double, 4, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Geometry/Quaternion.h:46:50:   required from 'class Eigen::QuaternionBase<Eigen::Quaternion<double> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Geometry/Quaternion.h:232:7:   required from 'class Eigen::Quaternion<double>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Geometry/arch/Geometry_SSE.h:92:3:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:55:30: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
                      >::type PacketReturnType;
                              ^~~~~~~~~~~~~~~~
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/SparseCore:37,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Sparse:26,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:14,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/SparseCore/SparseMatrixBase.h: In instantiation of 'class Eigen::SparseMatrixBase<Eigen::SparseMatrix<double, 0, int> >':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/SparseCore/SparseCompressedBase.h:36:7:   required from 'class Eigen::SparseCompressedBase<Eigen::SparseMatrix<double, 0, int> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/SparseCore/SparseMatrix.h:96:7:   required from 'class Eigen::SparseMatrix<double, 0, int>'
C:/R/R-4.0.2/library/RcppEigen/include/unsupported/Eigen/src/IterativeSolvers/ConstrainedConjGrad.h:61:25:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/SparseCore/SparseMatrixBase.h:47:30: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
                      >::type PacketReturnType;
                              ^~~~~~~~~~~~~~~~
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:434,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h: In instantiation of 'class Eigen::DenseCoeffsBase<Eigen::Matrix<int, -1, 1>, 0>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:300:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<int, -1, 1>, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:551:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<int, -1, 1>, 3>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseBase.h:41:34:   required from 'class Eigen::DenseBase<Eigen::Matrix<int, -1, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/MatrixBase.h:48:34:   required from 'class Eigen::MatrixBase<Eigen::Matrix<int, -1, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/PlainObjectBase.h:98:7:   required from 'class Eigen::PlainObjectBase<Eigen::Matrix<int, -1, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Matrix.h:178:7:   required from 'class Eigen::Matrix<int, -1, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/unsupported/Eigen/src/KroneckerProduct/KroneckerTensorProduct.h:168:29:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:55:30: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<int>::type' {aka '__vector(2) long long int'} [-Wignored-attributes]
                      >::type PacketReturnType;
                              ^~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h: In instantiation of 'class Eigen::DenseCoeffsBase<Eigen::Transpose<Eigen::Matrix<int, -1, 1> >, 0>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:300:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Transpose<Eigen::Matrix<int, -1, 1> >, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:551:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Transpose<Eigen::Matrix<int, -1, 1> >, 3>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseBase.h:41:34:   required from 'class Eigen::DenseBase<Eigen::Transpose<Eigen::Matrix<int, -1, 1> > >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/MatrixBase.h:48:34:   required from 'class Eigen::MatrixBase<Eigen::Transpose<Eigen::Matrix<int, -1, 1> > >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Transpose.h:115:37:   required from 'class Eigen::TransposeImpl<Eigen::Matrix<int, -1, 1>, Eigen::Dense>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Transpose.h:52:37:   required from 'class Eigen::Transpose<Eigen::Matrix<int, -1, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/unsupported/Eigen/src/KroneckerProduct/KroneckerTensorProduct.h:178:72:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:55:30: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<int>::type' {aka '__vector(2) long long int'} [-Wignored-attributes]
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h: In instantiation of 'class Eigen::DenseCoeffsBase<Eigen::Matrix<int, -1, -1>, 0>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:300:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<int, -1, -1>, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:551:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<int, -1, -1>, 3>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseBase.h:41:34:   required from 'class Eigen::DenseBase<Eigen::Matrix<int, -1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/MatrixBase.h:48:34:   required from 'class Eigen::MatrixBase<Eigen::Matrix<int, -1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/PlainObjectBase.h:98:7:   required from 'class Eigen::PlainObjectBase<Eigen::Matrix<int, -1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Matrix.h:178:7:   required from 'class Eigen::Matrix<int, -1, -1>'
C:/R/R-4.0.2/library/RcppEigen/include/unsupported/Eigen/src/KroneckerProduct/KroneckerTensorProduct.h:179:36:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:55:30: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<int>::type' {aka '__vector(2) long long int'} [-Wignored-attributes]
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:367,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h: In instantiation of 'struct Eigen::internal::find_best_packet<double, -1>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Matrix.h:22:57:   required from 'struct Eigen::internal::traits<Eigen::Matrix<double, -1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/PlainObjectBase.h:98:7:   required from 'class Eigen::PlainObjectBase<Eigen::Matrix<double, -1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Matrix.h:178:7:   required from 'class Eigen::Matrix<double, -1, -1>'
C:/R/R-4.0.2/library/Rcpp/include/Rcpp/InputParameter.h:51:11:   required from 'class Rcpp::ReferenceInputParameter<Eigen::Matrix<double, -1, -1> >'
RcppExports.cpp:16:73:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:44: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
          bool Stop = Size==Dynamic || (Size%unpacket_traits<PacketType>::size)==0 || is_same<PacketType,typename unpacket_traits<PacketType>::half>::value>
                                       ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
          bool Stop = Size==Dynamic || (Size%unpacket_traits<PacketType>::size)==0 || is_same<PacketType,typename unpacket_traits<PacketType>::half>::value>
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:170:83: warning: ignoring attributes on template argument 'Eigen::internal::unpacket_traits<__vector(2) double>::half' {aka '__vector(2) double'} [-Wignored-attributes]
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/util/XprHelper.h:188:88: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
   typedef typename find_best_packet_helper<Size,typename packet_traits<T>::type>::type type;
                                                                                        ^~~~
In file included from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Core:434,
                 from C:/R/R-4.0.2/library/RcppEigen/include/Eigen/Dense:1,
                 from C:/R/R-4.0.2/library/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:13,
                 from <command-line>:
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h: In instantiation of 'class Eigen::DenseCoeffsBase<Eigen::Matrix<double, -1, -1>, 0>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:300:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<double, -1, -1>, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:551:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<double, -1, -1>, 3>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseBase.h:41:34:   required from 'class Eigen::DenseBase<Eigen::Matrix<double, -1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/MatrixBase.h:48:34:   required from 'class Eigen::MatrixBase<Eigen::Matrix<double, -1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/PlainObjectBase.h:98:7:   required from 'class Eigen::PlainObjectBase<Eigen::Matrix<double, -1, -1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Matrix.h:178:7:   required from 'class Eigen::Matrix<double, -1, -1>'
C:/R/R-4.0.2/library/Rcpp/include/Rcpp/InputParameter.h:51:11:   required from 'class Rcpp::ReferenceInputParameter<Eigen::Matrix<double, -1, -1> >'
RcppExports.cpp:16:73:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:55:30: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
                      >::type PacketReturnType;
                              ^~~~~~~~~~~~~~~~
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h: In instantiation of 'class Eigen::DenseCoeffsBase<Eigen::Matrix<double, -1, 1>, 0>':
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:300:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<double, -1, 1>, 1>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:551:7:   required from 'class Eigen::DenseCoeffsBase<Eigen::Matrix<double, -1, 1>, 3>'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseBase.h:41:34:   required from 'class Eigen::DenseBase<Eigen::Matrix<double, -1, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/MatrixBase.h:48:34:   required from 'class Eigen::MatrixBase<Eigen::Matrix<double, -1, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/PlainObjectBase.h:98:7:   required from 'class Eigen::PlainObjectBase<Eigen::Matrix<double, -1, 1> >'
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/Matrix.h:178:7:   required from 'class Eigen::Matrix<double, -1, 1>'
C:/R/R-4.0.2/library/Rcpp/include/Rcpp/InputParameter.h:51:11:   required from 'class Rcpp::ReferenceInputParameter<Eigen::Matrix<double, -1, 1> >'
RcppExports.cpp:17:75:   required from here
C:/R/R-4.0.2/library/RcppEigen/include/Eigen/src/Core/DenseCoeffsBase.h:55:30: warning: ignoring attributes on template argument 'Eigen::internal::packet_traits<double>::type' {aka '__vector(2) double'} [-Wignored-attributes]
make: *** [C:/R/R-40~1.2/etc/i386/Makeconf:229: RcppExports.o] Error 1
ERROR: compilation failed for package 'quadprogpp'
* removing 'C:/R/R-4.0.2/library/quadprogpp'
Error: Failed to install 'quadprogpp' from GitHub:
  (converted from warning) installation of package ‘C:/Users/spinkney/AppData/Local/Temp/RtmpCKEsi0/file69bc43556a8c/quadprogpp_1.1-0.tar.gz’ had non-zero exit status
```
</details>

--- a/src/Makevars.win
+++ b/src/Makevars.win
@@ -4,4 +4,4 @@
 
 ## With R 3.1.0 or later, you can uncomment the following line to tell R to 
 ## enable compilation with C++11 (where available)
-USE_CXX1X = 
+USE_CXX14 = 

